### ISSUE TITLE
Expected date for 2.14.3.977 hack version?

### ISSUE THREAD

**Issue by MtbGot** [ddvk/remarkable-hacks#469]
Hi!
I am new I wanted to install the deck hack but my device is already upgraded to version 2.14.3.977. Do you think to launch soon his version?  Thank you!!!

**Comment by ddvk** [comment]
i'll try to do it by the end of the week

**Comment by MtbGot** [comment]
> i'll try to do it by the end of the week

Thank you!!!! 😍😍😍

**Comment by mattcwarren** [comment]
ddvl. Thanks so much for completing this update. the features that you include are great!

**Comment by samjliu** [comment]
ddvk, Thank you very much. Great work! All the new features are intuitive!

**Comment by Wortlaut** [comment]
ddvk, me too. That's a good work again  Thanks a lot

**Comment by MtbGot** [comment]
@ddvk 

THANK YOU!!!!!!!!!!!!!!!!!!!!!!!

I just installed the rm2 ddvk hack AND the stylus one, they are amazing!!!!
And it was really eas to do it! Thank you very much, I believed it was going to be harder without any coding experience!

I am playing now with it at it is way more easy than before!

**Comment by dstango** [comment]
Thanks for updating the hacks!!!!
yet .... I turned on updates, didn't find an update, changed machine-id, and now get offered 2.14.3.1005 ... :-/

**Comment by pstemper** [comment]
Yes, I am seeing 1005 also.  I guess I will wait for the next hack update.  (Can't live without these @ddvk!  You rock!)
PStemper

**Comment by leinad152** [comment]
Same here. Just started the automatic update because i had a chance and the hack was online for 977 - and now I'm stuck with 1005. And waiting for the update. Thank you, ddvk, for all the work and effort you're putting into this!
And yes, I did try to switch partitions. That worked, the downgrading didn't. Ah, well, so I will be more appreciative when the update ist there!

Cheers, leinad152

**Comment by ddvk** [comment]
ah, cmon

**Comment by martin-braun** [comment]
Just a quick tip for anyone who is on a version with hack. When you update, you see the version your device downloads. If it's not the expected version, you can just disable automatic updates to cancel the download and preserve your hack.

> ah, cmon

man I feel your pain.

**Comment by regrubtaf** [comment]
> Just a quick tip for anyone who is on a version with hack. When you update, you see the version your device downloads. If it's not the expected version, you can just disable automatic updates to cancel the download and preserve your hack. Then you just wait again, but at least you don't have to wait without hacks ...
> 
> > ah, cmon
> 
> man I feel your pain.

Similarly, if you accidentally leave auto-updating on and it downloads and installs a new version, the changes don't take effect until you restart the tablet.  You can just keep chugging along with the older version without restarting.

**Comment by ddvk** [comment]
i've added 1005

**Comment by leinad152** [comment]
Thank's a lot, that was quick! Works perfectly.

And thanks for the advise, regrubtaf and martin-braun, I'll try to remember that.

**Comment by pstemper** [comment]
Thank you so much @ddvk !!!

**Comment by dstango** [comment]
Thanks so much for updating it!
Might it be that the transparency for the suspended-image disappeared? It's only a minor cosmetic issue, but I wanted to report it anyhow ;-) ...

**Comment by dreadful-brain** [comment]
In the hacks for 1005 transparency in suspended png works

**Comment by dstango** [comment]
doesn't work for me on 1005. I'm on rm1. You?

**Comment by regrubtaf** [comment]
@dstango works for me in v.1005 on the rm2.

**Comment by dreadful-brain** [comment]
@dstango I‘m also on rM2

**Comment by dstango** [comment]
Okay, I found it half-way-working on my rm1: when going to suspend on itself transparency is working, when pressing the power button to go to suspend it's not working.
